### DISCUSSION TITLE
Petty Cash - Renaming the module

### DISCUSSION THREAD

**Discussion by andrewcouling** [timschofield/webERP#278] [Development Discussion & Specification]
18 years ago I worked in a Power Tools manufacturing business which operated a petty cash box, for staff expenses. Since then, having worked in a bunch of different companies, I have not encountered it again. In a new world of contactless payments, I wonder if the term Petty Cash is no longer relevant in business. May I suggest that we consider renaming the Petty Cash module to 'Expenses'? The term Petty Cash or abbreviation 'PC' occurs frequently throughout the various related transactions, so a full review of naming would be required. Thoughts?

**Comment by pakricard** [comment]
Hi @andrewcouling :

IMO: In my part of the world, "Petty Cash" is commonly used and "Expenses" is a broader term that is used for all expenses a company incurs. Paying the monthly Internet connection subscription via Bank is an expense, but not a petty cash expense, so my vote is to keep it as is.

Let's see the opinion of the rest of the community.

**Comment by timschofield** [comment]
This is a tricky one. I think the answer is that it depends very much where
you are in the world. I have certainly worked in places where "Petty Cash"
is still a term used for exactly this sort of system. I can also see that
there are places (such as the UK) where it might be deemed old fashioned. I
have been trying and failing to think of an alternative. Am seeing my son
later who is also an accountant, I will see what a younger accountant
thinks :)

If we can't find an alternative, as this is Ricard's module, I would say we
should probably leave as is.

Tim

On Fri, 6 Dec 2024 at 00:21, pakricard ***@***.***> wrote:

> Hi @andrewcouling <https://github.com/andrewcouling> :
>
> IMO: In my part of the world, "Petty Cash" is commonly used and "Expenses"
> is a broader term that is used for all expenses a company incurs. Paying
> the monthly Internet connection subscription via Bank is an expense, but
> not a petty cash expense, so my vote is to keep it as is.
>
> Let's see the opinion of the rest of the community.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/278#discussioncomment-11479182>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LH6MIEX67LIGUWHXO32EDUZHAVCNFSM6AAAAABTDSYKM2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBXHEYTQMQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by pakricard** [comment]
I guess this could be a never-ending discussion, as we all use different versions of English.

As example, some scripts use "Debtor", some "Customer", some "Stock code", some "Item code", some "inventory" and some "Stock" etc depending on the developer.

I think we can leave this issue post v5.0, and find a way to select these flavours of common names (probably in DB) so everybody is happy and using the expected words in their part of the world.

**Comment by timschofield** [comment]
Agreed.  For information, my son suggested "Staff Expenses" but that is
probably wrong as well.

Gettext would probably be the best way to resolve in future.

Thanks
Tim

On Mon, 9 Dec 2024 at 02:13, pakricard ***@***.***> wrote:

> I guess this could be a never-ending discussion, as we all use different
> versions of English.
>
> As example, some scripts use "Debtor", some "Customer", some "Stock code",
> some "Item code", some "inventory" and some "Stock" etc depending on the
> developer.
>
> I think we can leave this issue post v5.0, and find a way to select these
> flavours of common names (probably in DB) so everybody is happy and using
> the expected words in their part of the world.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/278#discussioncomment-11503056>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LEHXE3BCNRQFXRQ62L2ET4C7AVCNFSM6AAAAABTDSYKM2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNJQGMYDKNQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by pakricard** [reply]
Staff expenses would do the trick, as it differentiates regular expenses like invoices from suppliers, electricity bills, etc from "petty cash" expenses

**Comment by dalers** [comment]
I respectfully think the issue is more related to what the function of the module is, and less to what the popular name in a region is.

Imho (and IANAA - I am not an accountant ;-)  ), but I suspect the person who created the Petty Cash module was thinking that expenses would be reimbursed from a "petty cash box". However, I suspect in practise that many, if not most, reimbursements would be paid by cheque or added to employees next pay, unrelated to the "petty cash" concept.

I have not used the webERP "Petty Cash" module, but from the manual it seems the module provides for submitting, approving, reimbursing and reporting on employee expenses made on behalf of the business (i.e. _not_ the management of a physical petty cash box), in which case "Expenses" does seem to be a more appropriate name. However, I would argue what really should be done is to create a new "Human Resources" module with Expenses-related sub-menu items as well as the "Employees" sub-menu item from the Manufacturing module.

**Comment by timschofield** [comment]
Hi Dale, the module was originally created by Ricard, to fulfil a need in
his company. The functionality works well as it is designed. Basically a
float/advance is issued and authorised, expenses are returned to the
company together with any photos/scans of the receipts, which then need to
be authorised. The whole tab can then be reported on, showing how much of
the float/advance remains.
These days in the UK, most of this would be accomplished using company
credit cards, and this functionality would be done in the payables module.
In this case I would just hide the petty cash module. However in other
places I have worked, the system Ricard designed would work really well.

Tim

On Mon, 9 Dec 2024 at 19:58, Dale Scott ***@***.***> wrote:

> I respectfully think the issue is more related to what the function of the
> module is, and less to what the popular name in a region is.
>
> Imho (and IAMAC - I am not an accountant ;-) ), but I suspect the person
> who created the Petty Cash module was thinking that expenses would be
> reimbursed from a "petty cash box". I suspect in practise though that many,
> if not most, reimbursements would be paid by cheque or added to their
> employees next pay, unrelated to the "petty cash" concept.
>
> I have not used the webERP "Petty Cash" module, but from the manual it
> seems the module provides for submitting, approving, reimbursing and
> reporting on employee expenses (i.e. *not* the management of a physical
> petty cash box), in which case "Expenses" seems to be a more appropriate
> name. However, I would argue what really should be done is to create a new
> "Human Resources" module with Expenses-related sub-menu items as well as
> the "Employees" sub-menu from the Manufacturing module.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/278#discussioncomment-11512473>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LCKGQFQGC6OMWAJFC32EXY63AVCNFSM6AAAAABTDSYKM2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNJRGI2DOMY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by dalers** [reply]
> ...the system Ricard designed would work really well.

Agree the functionality looks very good. @pakricard can you describe what the term "Petty Cash" means to you?

Btw, I noticed the the built-in manual does not jump to the Petty Cash chapter. I'll see if I can fix this following [the instructions in the Wiki](https://github.com/timschofield/webERP/wiki/HelpSystem).

**Reply by pakricard** [reply]
The module was designed to control the small expenses paid by the employees that are repaid by the company, against a receipt. Fuel for the car, lunch with customer, etc. 
It works like a bank account between the company and each employee. Company can assign money to that account (give money to the employee) and authorize the expenses after someone revises them (and posting them automatically on the GL system).
It was written around 10 years ago, and the code can be improved. Also on my to-do list after v5.

**Reply by dalers** [reply]
@pakricard but _why_ do you call that functionality "Petty Cash"?  How do _you_ define "Petty Cash"?

**Reply by pakricard** [reply]
I'm not a native English speaker, so we called it “Petty Cash module” because it manages the “Petty Cash”, “Petty Cash” being the company money in the pocket of employees to pay for their daily small expenses that should be reimbursed by the company.

**Comment by andrewcouling** [comment]
Thank you all for your perspectives on this. I agree with and like with the suggestion in the following from @pakricard :

> I think we can leave this issue post v5.0, and find a way to select these flavours of common names (probably in DB) so everybody is happy and using the expected words in their part of the world.
